recipe/meta.yaml

```diff
@@ -36,10 +36,25 @@ requirements:
     - traitlets
 
 test:
+  requires:
+    - nose
+    - setuptools
+    - wheel
+    - nose
+    - coverage
+    - nodejs
+    - phantomjs
```

Comment:
> @pelson: are you :+1: to increasing test robustness, or not using CF as a CI?

conda-forge isn't a CI for projects. If you were to find project related issues on conda-forge, you would be finding the problems too late as they would already have been integrated into the codebase and tagged/released. The only motivation for running tests in a feedstock is to ensure that the built distribution is behaving as expected when installed. 

> I would maintain it probably does make sense as an integration testing platform

I completely agree with your motivation, but agree with @minrk that conda-forge probably isn't the place to do that unless the overhead (on you as developers, and on conda-forge's CI infrastructure and maintainability) is minimal. If the conda-forge setup lends well to being that place for full (read: long) integration testing, it is completely possible to set up a feedstock outside of conda-forge which you would register with the freely available CI services that conda-forge uses.